mediawiki123-apb/Dockerfile

```diff
@@ -23,8 +23,6 @@ ZGlhd2lraV9kYl9zY2hlbWEKICAtIG1lZGlhd2lraV9zaXRlX25hbWUKICAtIG1lZGlhd2lraV9z\
 aXRlX2xhbmcKICAtIG1lZGlhd2lraV9hZG1pbl91c2VyCiAgLSBtZWRpYXdpa2lfYWRtaW5fcGFz\
 cwo="
 
-
-ADD playbooks /opt/apb/actions
-ADD roles /opt/ansible/roles
+RUN yum -y install mediawiki-apb-role && yum clean all
```

Comment:
Could you create a `Dockerfile-dev` with the old behavior to make development/iteration easier for these?